client/src/store/new-request-module.ts

```diff
@@ -3368,6 +3369,11 @@ export class NewRequestModule extends VuexModule {
   mutateAnalyzePending (value: boolean) {
     this.analyzePending = value
   }
+
+  @Mutation
+  mutateIsLoadingSubmission (val: boolean) {
+    this.isLoadingSubmission = val
+  }
```

Comment:
Though I hate to add to this monstrous module, it is a suitable place to keep this state property, And BTW you have to use `@Mutation` for this -- a regular setter doesn't work.

I'm looking forward to replacing this module with a simple, global store/state object.

Reply:
Agrreeeeeeeeddddd